hugo/content/Upgrade/upgrade4xto42_bash.md

```diff
@@ -140,13 +157,13 @@ To ensure that you made any required configuration changes, you can compare with
     make deployoperator
 
 ##### Step 9
-The Operator is now upgraded to 4.1.0 and all users and roles have been recreated.
+The Operator is now upgraded to 4.2.0 and all users and roles have been recreated.
 Verify this by running:
 
     pgo version
 
 ##### Step 10
-Once the Operator is installed and functional, create new 4.1 clusters with the same name as was used previously. This will allow the new clusters to utilize the existing PVCs.
+Once the Operator is installed and functional, create a new 4.2.0 cluster with the same name and using the same major PostgreSQL version as was used previously. This will allow the new clusters to utilize the existing PVCs.
```

Comment:
```suggestion
Once the Operator is installed and functional, create a new 4.2.0 cluster with the same name and using the same major PostgreSQL version as was used previously. This will allow the new cluster to utilize the existing PVCs.
```

Reply:
Good catch, I'll make these updates.